.changeset/pre.json

```diff
@@ -21,7 +21,8 @@
     "@sveltejs/vite-plugin-svelte": "1.0.0-next.0",
     "@sveltejs/adapter-cloudflare-workers": "0.0.1",
     "set-env-action": "1.0.0",
-    "~TODO~": "0.0.1"
+    "~TODO~": "0.0.1",
```

Comment:
This seems like a potential mistake?

Reply:
I think it's just an artifact of the project template having that name — it's been there for a long time, it should be harmless